grpc/build.gradle

```diff
@@ -23,77 +10,4 @@ managedDependencies {
 
     testCompile 'io.grpc:grpc-okhttp'
     testCompile 'io.grpc:grpc-testing'
-
-    jmh 'io.grpc:grpc-okhttp'
-    if (upstreamJmh) {
-        jmh 'io.grpc:grpc-netty'
-    }
-}
-
-dependencies {
-    if (upstreamJmh) {
-        // Since project(':core') is automatically added in the top-level build, it's difficult to exclude
-        // its transitive dependencies just for upstream benchmarks. Fixing the versions to upstream works ok.
-        jmh 'com.google.guava:guava:19.0'
-    }
-}
-
-def generatedFilesDir = "$projectDir/gen-src"
-
-sourceSets {
-    main {
-        output.dir("${generatedFilesDir}/main/resources", builtBy: 'generateProto')
-    }
-    test {
-        output.dir("${generatedFilesDir}/test/resources", builtBy: 'generateTestProto')
-    }
-    jmh {
-        output.dir("${generatedFilesDir}/jmh/resources", builtBy: 'generateJmhProto')
-    }
-}
-
-protobuf {
```

Comment:
Guess we still need this protobuf task for generating `grpc/src/test/proto/com/linecorp/armeria/grpc/testing/`

Reply:
Oops - had forgotten to add the source sets, fixed